src/loader/load_sonar_dataset.py

```diff
@@ -42,7 +39,7 @@ def load_dataset(dataset_path: str, limit: int = None) -> "list[Recording]":
     if not os.path.exists(dataset_path):
         raise Exception("The dataset_path does not exist")
 
-    recordings: list[Recording] = []
+    recordings: "list[Recording]" = []
```

Comment:
This is not a forward reference, so the types shouldn't be in quotes.

Reply:
(https://peps.python.org/pep-0484/#forward-references)

Reply:
Thanks